cmd/uplink/cmd/root.go

```diff
@@ -109,10 +105,10 @@ func (c *UplinkFlags) GetProject(ctx context.Context) (*libuplink.Project, error
 }
 
 // GetProjectAndBucket returns a *libuplink.Bucket for interacting with a specific project's bucket
-func (c *UplinkFlags) GetProjectAndBucket(ctx context.Context, bucketName string, access libuplink.EncryptionAccess) (project *libuplink.Project, bucket *libuplink.Bucket, err error) {
-	project, err = c.GetProject(ctx)
+func (cliCfg *UplinkFlags) GetProjectAndBucket(ctx context.Context, bucketName string, access libuplink.EncryptionAccess) (project *libuplink.Project, bucket *libuplink.Bucket, err error) {
+	project, err = cliCfg.GetProject(ctx)
 	if err != nil {
-		return nil, nil, err
+		return project, bucket, err
```

Comment:
I have seen pre-defined return values in other places in the code. Is there a convention that we would always return the pre-defined variables vs we would be explicit to return `nil`?

Reply:
im not sure if there is a convention. Personally I just like using the name of the return value if it has a name since its more clear what "nil" is for and saves me a step trying to figure out what nil is suppose to be